fiftyone/core/stages.py

```diff
@@ -5205,26 +5220,122 @@ class SelectFields(ViewStage):
             May contain ``embedded.field.name`` as well
     """
 
-    def __init__(self, field_names=None, _allow_missing=False):
+    def __init__(
+        self, field_names=None, _allow_missing=False, _meta_filter=None
```

Comment:
By convention, arg names for `ViewStage` constructor should match the corresponding `@view_stage` method on `SampleCollection`.

So here, go ahead and use `meta_filter`. And copy the docstring changes over here 😄